.github/workflows/dbt-checks.yaml

```
@@ -0,0 +1,53 @@
+      - name: Setup DBT profile
+        working-directory: queries
+        run: sh setup_dbt_profiles.sh
+
+      - name: Run DBT compile
+        working-directory: queries
+        run: |-
+          dbt compile --profiles-dir profiles
+
+      - name: Run sqlfmt
+        working-directory: queries
+        run: sqlfmt --diff ${{ steps.changed-files.outputs.all_changed_files }}
```

actionlint (1.7.7)

property "changed-files" is not defined in object type {}

(expression)